bindata/network/ovn-kubernetes/self-hosted/common/004-config.yaml

```diff
@@ -26,6 +26,9 @@ data:
 {{- if .OVNHybridOverlayEnable }}
     no-hostsubnet-nodes="kubernetes.io/os=windows"
 {{- end  }}
+{{- if .IsNetworkTypeMigration }}
+    no-hostsubnet-nodes="migration.network.openshift.io/plugin="
```

Comment:
What if OVNHybridOverlayEnable is also true?
Is no-hostsubnet-nodes additive?

Reply:
We don't support rollback a HO-enabled cluster. Since OpenShiftSDN doesn't support HO. So these conditions cannot be true at the same time.